src/lection2/Main.java

```diff
@@ -0,0 +1,30 @@
+package lection2;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        double sum = 0;
+        int numberSubjects;
+        double averageMark;
+        Scanner Numscanner = new Scanner(System.in);
+
+        System.out.println("Введите количество предметов: ");
+
+        numberSubjects = Numscanner.nextInt();
+
+        for (int x = 1; x <= numberSubjects; x++) {
+            System.out.println("\nВведите оценку за " + x + " предмет:");
+            int mark = Numscanner.nextInt();
+            if (mark > 10 || mark <= 0) {
+
+                System.out.println("Ошибка! Введите оценку от 1 до 10.");
```

Comment:
<img width="315" alt="Снимок экрана 2024-01-07 в 14 30 29" src="https://github.com/vjpictures/Lection-2/assets/73035222/58129156-78b5-4065-8b2d-89e0b4d4d29c">

Тут программа должна закончить свое выполнение

Reply:
Подсказка - напиши `return;` после System.out.....

return  прекращает выполнение функции  main и программы в целом, т.к. у нас программа состоит только из этой функции

После строки
```suggestion
                System.out.println("Ошибка! Введите оценку от 1 до 10.");
                return;
```